SparkConnector/setup.py

```diff
@@ -69,6 +69,7 @@
     install_requires=[
         "jupyterlab~=3.0",
         "bs4",
+        "swanportallocator",
```

Comment:
You might need to move `jupyter-config/` from `data_files_spec` into the new extension

Reply:
Missed that, thanks!